tensorflow/examples/tutorials/mnist/mnist_rnn.py

```diff
@@ -0,0 +1,120 @@
+""" Recurrent Neural Network.
+A Recurrent Neural Network (LSTM) implementation example using TensorFlow library.
+This example is using the MNIST database of handwritten digits (http://yann.lecun.com/exdb/mnist/)
+Links:
+    [Long Short Term Memory](http://deeplearning.cs.cmu.edu/pdfs/Hochreiter97_lstm.pdf)
+    [MNIST Dataset](http://yann.lecun.com/exdb/mnist/).
+"""
+
+from __future__ import print_function
```

Comment:
Add the other futures (`division`, `absolute_import`).

Reply:
Thanks, added.